src/shared/Microsoft.Git.CredentialManager/CommandContext.cs

```diff
@@ -112,52 +95,7 @@ public CommandContext()
 
         public ISettings Settings { get; }
 
-        public TextReader StdIn
-        {
-            get
-            {
-                if (_stdIn == null)
-                {
-                    _stdIn = new StreamReader(Console.OpenStandardInput(), Utf8NoBomEncoding);
-                }
-
-                return _stdIn;
-            }
-        }
-
-        public TextWriter StdOut
-        {
-            get
-            {
-                if (_stdOut == null)
-                {
-                    _stdOut = new StreamWriter(Console.OpenStandardOutput(), Utf8NoBomEncoding)
-                    {
-                        AutoFlush = true,
-                        NewLine = LineFeed,
-                    };
-                }
-
-                return _stdOut;
-            }
-        }
-
-        public TextWriter StdError
-        {
-            get
-            {
-                if (_stdErr == null)
-                {
-                    _stdErr = new StreamWriter(Console.OpenStandardError(), Utf8NoBomEncoding)
-                    {
-                        AutoFlush = true,
-                        NewLine = LineFeed,
-                    };
-                }
-
-                return _stdErr;
-            }
-        }
+        public IStandardStreams Streams { get; }
```

Comment:
There are many changes for the Streams update.  I really like that you put the changes for this in it's own commit with details.  If you hadn't nicely done that I think would have asked for a separate PR.  But since you did go you :-)!